### PR TITLE
article updates

### LINKED ISSUE #11381: Opening VS Cmd Promt from with VS - Initial directory field, such as Project Directory
this is good info to say that we probably want to set the initial directory as Project Directory, but some more information on how to set the directory as project directory would be helpful.  I don't know how to do that currently.  

Thanks, 
Greg

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0a1428fb-1282-c537-8714-29cf2ccb0446
* Version Independent ID: d329fbf9-0b17-e4e4-733f-9b9c96eb46bd
* Content: [Developer Command Prompt for Visual Studio](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs#feedback)
* Content Source: [docs/framework/tools/developer-command-prompt-for-vs.md](https://github.com/dotnet/docs/blob/master/docs/framework/tools/developer-command-prompt-for-vs.md)
* Product: **dotnet-framework**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### LINKED ISSUE #15611: Review article - CSAT OKR
Article needs to be reviewed because of low CSAT scores.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0a1428fb-1282-c537-8714-29cf2ccb0446
* Version Independent ID: d329fbf9-0b17-e4e4-733f-9b9c96eb46bd
* Content: [Developer Command Prompt for Visual Studio](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
* Content Source: [docs/framework/tools/developer-command-prompt-for-vs.md](https://github.com/dotnet/docs/blob/master/docs/framework/tools/developer-command-prompt-for-vs.md)
* Product: **dotnet-framework**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #15611 
Fixes #11381

### CODE REVIEW

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 1]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
 ### In Windows 10
 
-1. In the search box on the taskbar, start typing the name of the tool, such as `dev` or `developer command prompt`. This brings up a list of installed apps that match your search pattern. If you're looking for a different command prompt, try entering a different search term such as `prompt`.
+1. Select **Start** ![Windows logo key on the keyboard.](./media/developer-command-prompt-for-vs/windows-logo-key-graphic.png) and scroll to the letter **V**.
 
-2. Choose the **Developer Command Prompt for Visual Studio** (or the command prompt you want to use).
+1. Expand the **Visual Studio 2019** folder.
 
-### In Windows 8.1
+1. Choose the **Developer Command Prompt for VS 2019** (or the command prompt you want to use).
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
1. Choose **Developer Command Prompt for VS 2019** (or the command prompt you want to use).
```

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 2]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
 
-3. Choose the **Developer Command Prompt** (or the command prompt you want to use).
-
-### In Windows 8
+### In Windows 8.1
 
 1. Go to the **Start** screen, by pressing the Windows logo key ![Windows logo key on the keyboard.](./media/developer-command-prompt-for-vs/windows-logo-key-graphic.png) on your keyboard for example.
 
-2. On the **Start** screen, press the Windows logo key ![Windows logo key on the keyboard.](./media/developer-command-prompt-for-vs/windows-logo-key-graphic.png) `+ Z`.
+1. On the **Start** screen, press **Ctrl**+**Tab** to open the **Apps** list, and then enter `V`. This brings a list that includes all installed Visual Studio command prompts.
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
1. On the **Start** screen, press **Ctrl**+**Tab** to open the **Apps** list, and then press **V**. This brings up a list that includes all installed Visual Studio command prompts.
```

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 3]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
 
-3. Choose the **Apps view** icon at the bottom of the screen and then enter `V`. This brings a list that includes all installed Visual Studio command prompts.
-
-4. Choose the **Developer Command Prompt** (or the command prompt you want to use).
+1. Choose the **Developer Command Prompt for VS 2019** (or the command prompt you want to use).
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
1. Choose **Developer Command Prompt for VS 2019** (or the command prompt you want to use).
```

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 4]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
 
 ## Search for the command prompt on your machine
 
-You may have multiple command prompts, depending on the version of Visual Studio and any additional SDKs you've installed. For example, 64-bit versions of Visual Studio provide both 32-bit and 64-bit command prompts. (The 32-bit and 64-bit versions of most tools are the same; however, a few tools make changes specific to 32-bit and 64-bit environments.) If the following steps don't work, you can try [Manually locate the files on your machine](#manually-locate-the-files-on-your-machine) or [Run the command prompt from inside Visual Studio](#run-the-command-prompt-from-inside-visual-studio).
+You may have multiple command prompts, depending on the version of Visual Studio and any additional SDKs and workloads you've installed. If the following steps don't work, you can try to [manually locate the files on your machine](#manually-locate-the-files-on-your-machine) or [start the command prompt from inside Visual Studio](#start-the-command-prompt-from-inside-visual-studio).
 
 ### In Windows 10
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
### Windows 10
```

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 5]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
-1. Go to the **Start** screen, by pressing the Windows logo key ![Windows logo key on the keyboard.](./media/developer-command-prompt-for-vs/windows-logo-key-graphic.png) on your keyboard for example.
+   Alternatively, you can start typing the name of the command prompt in the search box on the taskbar, and choose the result you want as the result list starts to display the search matches.
 
-2. On the **Start** screen, press **Ctrl**+**Tab** to open the **Apps** list, and then enter `V`. This brings a list that includes all installed Visual Studio command prompts.
+   ![Animated gif showing the search behavior on Windows 10](./media/developer-command-prompt-for-vs/windows10-search.gif)
 
-3. Choose the **Developer Command Prompt** (or the command prompt you want to use).
-
-### In Windows 8
+### In Windows 8.1
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
### Windows 8.1
```

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 6]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
 
-2. On the **Start** screen, press the Windows logo key ![Windows logo key on the keyboard.](./media/developer-command-prompt-for-vs/windows-logo-key-graphic.png) `+ Z`.
+1. On the **Start** screen, press **Ctrl**+**Tab** to open the **Apps** list, and then enter `V`. This brings a list that includes all installed Visual Studio command prompts.
 
-3. Choose the **Apps view** icon at the bottom of the screen and then enter `V`. This brings a list that includes all installed Visual Studio command prompts.
-
-4. Choose the **Developer Command Prompt** (or the command prompt you want to use).
+1. Choose the **Developer Command Prompt for VS 2019** (or the command prompt you want to use).
 
 ### In Windows 7
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
### Windows 7
```

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 7]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
+Usually, the shortcuts for the command prompts you have installed are placed at the **Start Menu** folder for Visual Studio, such as in *%ProgramData%\Microsoft\Windows\Start Menu\Programs\Visual Studio 2019\Visual Studio Tools*. But if for some reason, searching for the command prompt doesn't bring the expected results, you can try to manually locate the shortcut on your machine. Try searching for the name of the command prompt file, such as *VsDevCmd.bat*, or go to the Tools folder such as *%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Community\Common7\Tools* (path changes according to your Visual Studio version, edition, and installation location).
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
Usually, the shortcuts for the command prompts you have installed are placed at the **Start Menu** folder for Visual Studio, such as in *%ProgramData%\Microsoft\Windows\Start Menu\Programs\Visual Studio 2019\Visual Studio Tools*. But if, for some reason, searching for the command prompt doesn't produce the expected results, you can try to manually locate the shortcut on your machine. Try searching for the name of the command prompt file, such as *VsDevCmd.bat*, or go to the Tools folder, such as *%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Community\Common7\Tools* (path changes according to your Visual Studio version, edition, and installation location).
```

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 8]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
 
-## Run the command prompt from inside Visual Studio
+## Start the command prompt from inside Visual Studio
 
-For easier access, you can add the Visual Studio Developer Command Prompt, or any other command prompt, to the **Tools** menu in Visual Studio. To make the tool available, add it to the external tools list. Here are the steps:
+For easier access, you can add the Developer Command Prompt, or any other command prompt, to the **Tools** menu in Visual Studio. To make the tool available, add it to the external tools list. Here are the steps:
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
For easier access, you can add Developer Command Prompt, or any other command prompt, to the Tools menu in Visual Studio. To make the tool available, add it to the external tools list. Here are the steps:
```
I think we're only supposed to bold UI items when they're part of steps to follow.

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 9]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
+1. On the **External Tools** dialog box, choose the **Add** button. A new entry appears.
 
-3. On the **External Tools** dialog box, choose the **Add** button. A new entry appears.
+1. Enter a **Title** for your new menu item such as `Command Prompt`.
 
-4. Enter a **Title** for your new menu item such as `Command Prompt`.
+1. In the **Command** field, specify the file you want to launch, such as `%comspec%` or `C:\Windows\System32\cmd.exe`.
 
-5. In the **Command** field, specify the file you want to launch, such as `%comspec%` or `C:\Windows\System32\cmd.exe`.
+1. In the **Arguments** field, specify where to find the specific command prompt you want to use such as `/k "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\Tools\VsDevCmd.bat"` (this command launches the Developer Command Prompt that is installed with Visual Studio 2019 Community). Change this value according to your Visual Studio version, edition, and installation location.
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
1. In the **Arguments** field, specify where to find the specific command prompt you want to use, such as `/k "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\Tools\VsDevCmd.bat"`. This command launches the Developer Command Prompt that's installed with Visual Studio 2019 Community. Change this value according to your Visual Studio version, edition, and installation location.
```

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 10]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
 
-6. In the **Arguments** field, specify where to find the specific command prompt you want to use such as `/k "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat"` (this command launches the Developer Command Prompt that is installed with Visual Studio 2017 Enterprise). Change this value according to your Visual Studio version, edition, and installation location.
+1. In the **Initial directory** field, specify in which directory the command prompt will start. Choose a value such as **Project Directory** by selecting the arrow right next to the field.
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
1. In the **Initial directory** field, specify the directory in which the command prompt will start. Choose a value such as **Project Directory** by selecting the arrow next to the field.
```

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 11]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
...
 
-7. Choose a value for the **Initial directory** field, such as **Project Directory**.
+1. Choose the **OK** button.
 
-8. Choose the **OK** button.
+   ![External Tools dialog with the field values filled out.](./media/developer-command-prompt-for-vs/add-external-tool.png)
 
    The new menu item is added, and you can access the command prompt from the **Tools** menu.
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
   The new menu item is added, and you can access the command prompt from the Tools menu.
```

File: docs/framework/tools/developer-command-prompt-for-vs.md [hunk 12]
```diff
@@ -10,81 +10,83 @@ helpviewer_keywords:
   - "environment variables, setting for tools"
   - "developer command prompt"
 ms.assetid: 94fcf524-9045-4993-bfb2-e2d8bad44219
-author: "mairaw"
-ms.author: "mairaw"
 ---
 # Developer Command Prompt for Visual Studio
 
-The Developer Command Prompt for Visual Studio enables you to use .NET Framework tools more easily. It is a command prompt that automatically sets specific environment variables.
+The Developer Command Prompt for Visual Studio enables you to use .NET Framework tools more easily. It is a command prompt that automatically sets specific environment variables. After opening the Developer Command Prompt, you can start typing the commands for the [.NET Framework tools](index.md) such as `ildasm` or `clrver`.
```
Comment (gewarren) [inline docs/framework/tools/developer-command-prompt-for-vs.md]:
```suggestion
Developer Command Prompt for Visual Studio enables you to use .NET Framework tools more easily. It's a command prompt that automatically sets specific environment variables. After opening Developer Command Prompt, you can enter the commands for [.NET Framework tools](index.md) such as `ildasm` or `clrver`.
```

**Review by gewarren — APPROVED** [review]
Looks good. I left some suggestions for you to consider. I think it sounds better without "the" in front of "Developer Command Prompt" since it's a branded name for the app.

**Comment by mairaw** [comment]
Thanks for your review @gewarren. I've applied all your suggestions.

**Pull request merged**